.grunt-config/sass.js

```diff
@@ -88,7 +88,7 @@ const sass = {
 				expand: true,
 				cwd: 'assets/dev/scss/admin',
 				src: 'wc-product-editor.scss',
-				dest: 'assets/css',
+				dest: 'assets/css/conditionals',
```

Comment:
Why change the location of the WC Product Editor in this PR?

Reply:
@Batsirai-muchareva I think this is good point.
It is not good practice to change unrelated aspects in a PR.